packages/klevu-ui/src/components/klevu-quicksearch/klevu-quicksearch.tsx

```diff
@@ -314,12 +330,22 @@ export class KlevuQuicksearch {
       if (this.tTrendingCaption === undefined)
         this.tTrendingCaption =
           settings?.klevu_uc_userOptions.showTrendingProductsCaption ?? getTranslation("quicksearch.tTrendingCaption")
-      if (this.tPopularProductsTitle === undefined)
-        this.tPopularProductsTitle =
+      if (this.showRecentSearches === undefined) this.showRecentSearches = settings?.klevu_showRecentSerches
```

Comment:
We should add brackets to all of these single if's

Reply:
added